Add Web Payments collection to /learn (#2907)
GoogleChrome/web.dev: src/site/content/en/payments/empowering-payment-apps-with-web-payments/index.md

---
layout: post
title: Empowering payment apps with Web Payments
authors:
  - agektmr
description: |
  Web Payments evolves user's payment experience on the web. Learn how it works, its benefits and get ready to integrate your payment app with Web Payments.
date: 2020-05-25
---

One of the key ecosystem drivers for the web are payments. With secure,
seamless, and flexible payment systems, the web can become a sustainable and

> profitable platform. We believe that the Web Payments standards will be a key

Reviewer: ```suggestion
profitable platform. The Web Payments standards has the potential to be a key
```
We can avoid the problem with this rephrase

with browsers easier than ever.

The standards are flexible; they work with various types of payment systems and
are intended to work on any browser on any device, payment method, or payment
service provider. This flexibility enables development simplicity, deployment
consistency, and future compatibility with emerging payment technologies.

Research shows that [long checkout flows lead to cart
abandonment](https://baymard.com/blog/checkout-flow-average-form-fields). With
Web Payments, checkout flow is simplified to a few taps instead of manual entry
of billing data for every purchase. See a demo below of how Google Pay leverages
Web Payments to build a seamless flow. The same can be achieved by any other
payment apps:

  <figure class="w-figure">
    <video controls autoplay loop muted class="w-screenshot">
      <source src="https://storage.googleapis.com/web-dev-assets/payments/native-payment-app.webm" type="video/webm">
      <source src="https://storage.googleapis.com/web-dev-assets/payments/native-payment-app2.mp4" type="video/mp4">
    </video>
   <figcaption class="w-figcaption">Checkout flow with Google Pay and Web Payments.</a>.
    </figcaption>

>   </figure>

Author: I added it on purpose. Do you have a good idea how to fit this vertically long video fit into vertically long article?

>   </figure>

Author: Replaced the video with one without the backdrop. Added a custom style with fixed width + centering, but ideally there's a shared class to do the same. This is a very common style I wish there was. ex: https://web.dev/web-otp/

Working with payment gateways is also a good option as they can help scale your
outreach as well.

##  How much does it cost?

Web Payments is all about standard technology in the browser. Payment apps
adopting it nor activating it on the browser won't charge them any fees by
itself.

##  Browser support

Web Payments consists of a few different pieces of technologies and the support
status depends on the browser.

<table>
  <tr>
    <td></td>
    <td>Chrome (Chromium)</td>
    <td></td>
    <td></td>
    <td>Safari</td>
    <td></td>
    <td>Firefox</td>
  </tr>
  <tr>
    <td></td>
    <td>Desktop</td>
    <td>Android</td>
    <td>iOS</td>
    <td>Desktop</td>
    <td>Mobile</td>
    <td></td>
  </tr>
  <tr>
    <td>Payment Request API</td>
    <td>✔</td>
    <td>✔</td>
    <td></td>
    <td>✔</td>
    <td>✔</td>
    <td>Under active development</td>
  </tr>
  <tr>
    <td>Payment Handler API</td>
    <td>✔</td>
    <td>✔</td>
    <td></td>
    <td></td>
    <td></td>
    <td>Under active development</td>
  </tr>
  <tr>
    <td>Native payment app</td>
    <td>✔</td>
    <td>✔</td>
    <td>*</td>
    <td>✔**</td>
    <td>✔**</td>
    <td></td>
  </tr>

> </table>

Reviewer 2: My bad, I forgot that we have table markup and it looked fine on the big screen.
